binder/environment.yml

```diff
@@ -11,6 +11,7 @@ dependencies:
   - s3fs
   - gcsfs
   - appdirs
+  - packaging
```

Comment:
doesn't python provide default version management class (should come with either setuptools or the old distutils)